src/SWP/Bundle/ContentBundle/Form/Type/RouteToArticleType.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\ContentBundle\Form\Type;
+
+use SWP\Bundle\ContentBundle\Validator\Constraints\ArticleRule;
+use SWP\Bundle\ContentBundle\Validator\Constraints\RouteId;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class RouteToArticleType extends AbstractType
+{
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array                $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('rule', TextType::class, [
+                'required' => false,
+                'constraints' => [
+                    new ArticleRule(),
+                ],
+            ])
+            ->add('priority')
```

Comment:
As far as I understand it, this is the default, if priority is defined as an integer in the ORM mapping